build/ci/linux/package.sh

```diff
@@ -54,20 +54,23 @@ if [ "$BUILD_MODE" == "stable_build" ]; then PACKTYPE=appimage; fi
 
 MAJOR_VERSION="${BUILD_VERSION%%.*}"
 
+if [ -z "$PACKARCH" ]; then PACKARCH="x86_64"; fi
```

Comment:
Where do we get the `PACKARCH` variable from here?
If it is from environment variables, then it is better to pass it explicitly as an argument for these scripts (in order to be able to use them in other cases and be uniform)